src/server/src/velvet_dawn/dao/__init__.py

```diff
@@ -2,9 +2,29 @@
 
 from flask import Flask
 from flask_sqlalchemy import SQLAlchemy
+from pathlib import Path
+from os import remove
 
 import velvet_dawn
 
+db_path=Path(__file__).parent.parent.parent / "instance" / "game.db"
```

Comment:
I'm not keep on this since it requires user input. I want to be able to run this distributed so it'll need to run on headless server instances which wont accept user input

Furthermore, this work based on the premise that there is a game.db file. Whilst this is currently, true It wont be in future (you should be able to connect it to a remote instance).

A preferred solution would be to add another command in the velvet-dawn.sh file 'clean' and 'clean-and-run' (or what ever name you decide). This has the benefit that it would only be run by the user and not automatically on serverless infrastructure.

Reply:
The reason for for requiring it be distributed is so that it can scale up, for example, I want to be able to play with my team which will most likely be too large to the built in flask server to handle so will need to horizontally scale the infrastructure

Reply:
I would actually go one step further on this.

Since if use a remote db, you wont have a db file. Instead you should call the db.drop_all() function. So perhaps make another file in the dao, called clean.py. This has a __main__ == '__name__' function which calls db.drop_all(). Then you'll still be able to wipe a remote instance

Reply:
Have changed this to operate from velvet_dawn/dao/clean.py, only when '-c' is (optionally) entered at server launch.
User input prompt has been removed.